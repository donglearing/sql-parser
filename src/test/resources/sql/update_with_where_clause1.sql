update T set a = NULL where b = 1