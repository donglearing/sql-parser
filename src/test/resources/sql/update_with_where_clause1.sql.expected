UPDATE T SET a = NULL WHERE b = 1